packages/ng/form-field/input.directive.ts

```diff
@@ -0,0 +1,12 @@
+import { Directive, ElementRef, HostBinding, inject } from '@angular/core';
+
+@Directive({
+	selector: '[luInput]',
+	standalone: true,
+})
+export class InputDirective {
+	@HostBinding('class')
+	clazz = 'textField-input-value';
+
+	public host = inject<ElementRef<HTMLElement>>(ElementRef<HTMLElement>);
```

Comment:
No need to specify the second generic:

```suggestion
	public host = inject<ElementRef<HTMLElement>>(ElementRef);
```